docs/metrics/prometheus/README.md

```diff
@@ -81,7 +87,7 @@ If Prometheus endpoint authentication type is set to `public`. Following prometh
 ```yaml
 scrape_configs:
 - job_name: minio-job
-  metrics_path: /minio/prometheus/metrics
+  metrics_path: /minio/prometheus/metrics/cluster
```

Comment:
We should add the other endpoint /minio/prometheus/metrics/node as a separate job.

Reply:
Yes, mc needs a PR too. I don't know if we should prescribe using /node and just selectively move metrics over.

Reply:
IMHO for the sake of completion document should show all the endpoints.

Reply:
ok, will add it